zarr/tests/test_storage.py

```diff
@@ -721,6 +721,14 @@ def test_filesystem_path(self):
         store['foo'] = b'bar'
         assert os.path.isdir(path)
 
+        # check correct permissions
+        stat = os.stat(path)
+        mode = stat.st_mode & 0o666
+        umask = os.umask(0)
+        os.umask(umask)
+        assert mode == (0o666 & ~umask)
+
+
```

Comment:
This extra blank line is causing a `flake8` [linting error](https://travis-ci.org/zarr-developers/zarr-python/jobs/553953573#L2486). Removing the extra line should fix the failing Travis CI build:

```suggestion
```

Reply:
> This extra blank line is causing a `flake8` [linting error](https://travis-ci.org/zarr-developers/zarr-python/jobs/553953573#L2486). Removing the extra line should fix the failing Travis CI build:

Thanks for the heads up. Fixed now.